### DISCUSSION TITLE
Eclipse WindowBuilder change font size in pallette

### DISCUSSION THREAD

**Discussion by osamaaweek1** [eclipse/windowbuilder#603] [Q&A]
I need to change the palette text size because it is too small for my eyes.

i tried to do it through the properties but it always goes back to the same size. i even change the whole font through colors and fonts in the preferences but same result.

**Comment by ptziegler** [comment]
I agree, the current dialog is not intuitive at all.

If you right-click on the palette and select "Settings" in the context menu, it should show you the palette settings dialog.
There you can adjust the category and entry font.

Once you click "OK", you need to close and re-open the editor, before the changes are applied. But is just doesn't feel very intuitive and the fonts are all squished together and don't look very good overall.

The good news: The mechanism itself seems to be working.

![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/93305a6c-8043-4ed9-862c-e8d3a5459709)
